region/brinstar/blue.json

```
@@ -1422,6 +1384,50 @@
             {
               "id": 1,
               "strats": [
+                {
+                  "name": "Leave With Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 3,
+                      "openEnd": 1
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Charged",
+                  "notable": false,
+                  "requires": [
+                    {"canShineCharge": {
+                      "usedTiles": 31,
+                      "openEnd": 0
+                    }},
+                    "canShinechargeMovement" 
+                  ],
+                  "exitCondition": {
+                    "leaveCharged": {
+                      "framesRemaining": 120
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Charged - Power Bomb Blocks Broken",
+                  "notable": false,
+                  "requires": [
+                    {"obstaclesCleared": ["C"]},
+                    {"canShineCharge": {
+                      "usedTiles": 33,
+                      "openEnd": 0
```

With the new ways to calculate runway lengths, 33 tiles and 0 open ends won't be in logic if the player selects 33 tiles. We should eventually go through and increase the number of tiles a bit, maybe to 35 or 36, because 33 can be a bit tight for beginners.
```suggestion
                      "openEnd": 1
```

---

I can see that; being able to stop when you turn blue is hard at any point before you can 3 tap.  Knowing when to stop should come with a visual indicator that is the end of the runway.  Converting to spinjump if you run into a wall would help to have more runway leeway... and properly storing the shinespark is difficult early on.

`usedTiles` in schema should be updated then.   In this case you could fit an openend on both sides of a 33 tile runway but it is clearer if the usedTile count is increased instead.